language-plutus-core/src/Language/PlutusCore/Rename/Internal.hs

```diff
@@ -154,8 +56,8 @@ renameTypeM ty@TyBuiltin{}              = pure ty
 
 -- | Rename a 'Term' in the 'RenameM' monad.
 renameTermM
-    :: (HasUnique (tyname ann) TypeUnique, HasUnique (name ann) TermUnique)
-    => Term tyname name ann -> ScopedRenameM (Term tyname name ann)
+    :: (HasUniques (Term tyname name ann), MonadQuote m)
```

Comment:
Shortened

```haskell
 (HasUnique (tyname ann) TypeUnique, HasUnique (name ann) TermUnique)
```

to 

```haskell
HasUniques (Term tyname name ann)
```

which makes things more readable in a quite few places.